### ISSUE TITLE
Low accuracy on the validation set

### ISSUE THREAD

**Issue by FlYWMe** [cvcode18/imbalanced_learning#16]
Hi! Thanks for providing the codes !
When I was training, the train_mAP kept rising. However, the val_mAP can only be up to 0.80 at 16 epoch, and then it starts to decline, the overfitting is serious. The final saved model was the result of training to 16 epochs. Although the test set also reached 0.83, the training of leftover 200 epochs did not improve on the validation set leads to the model was not updated.

Thanks a lot!
